README.md

```diff
@@ -1,4 +1,4 @@
-# DCOS Signal Service [![Build Status](https://jenkins.mesosphere.com/service/jenkins/buildStatus/icon?job=dcos-cluster-ops/public-dcos-signal-service-master)](https://jenkins.mesosphere.com/service/jenkins/job/dcos-cluster-ops/job/public-dcos-signal-service-master/)
+# DCOS Signal Service [![Build Status](https://jenkins.mesosphere.com/service/jenkins/buildStatus/icon?job=public-dcos-cluster-ops/dcos-signal/add-standard-jenkinsfile)](https://jenkins.mesosphere.com/service/jenkins/job/public-dcos-cluster-ops/job/dcos-signal/job/add-standard-jenkinsfile/)
```

Comment:
Will we stick with that job name? That sounds a bit odd.

Reply:
Thanks, fixed. On master this job was not run but it will be after merging